src/stores/obsPortalData.js

```diff
@@ -0,0 +1,59 @@
+import { defineStore } from 'pinia'
+import { fetchApiCall } from '../utils/api'
+import { useUserDataStore } from './userData'
+import { useConfigurationStore } from './configuration'
+
+const fifteenDaysAgo = new Date(new Date().getTime() - 15 * 24 * 60 * 60 * 1000).toISOString().split('T')[0]
+
+export const useObsPortalDataStore = defineStore('obsPortalData', {
+  state () {
+    return {
+      completedObservations: {},
+      upcomingRealTimeSessions: {},
+      pendingRequestGroups: []
+    }
+  },
+  persist: true,
+  actions: {
+    sortResponseData (response) {
+      const sixteenMinutes = 16 * 60 * 1000
```

Comment:
Can we just add a comment as to why we're using 16 minutes here and what it's for - especially if we're not going to add a global configuration item for the session length? (we should do that at some point).